src/main/webapp/college-info.js

```diff
@@ -24,6 +24,18 @@ function init() {
  * Loads the college data from an object passed in local storage.
  */
 function loadSchoolInfo() {
+  const infoDivId = 'info';
```

Comment:
I think I'd rather see you store only the references to the elements themselves, and pass those elements into `toggleElementsDisplay` instead of requiring another DOM query.